e2e/cypress/support/api/members.ts

```diff
@@ -35,21 +35,21 @@ export function ensureHumanIsOrgMember(api: API, username: string, roles: string
 
 export function ensureHumanIsNotProjectMember(
   api: API,
-  projectId: string,
+  projectId: number,
```

Comment:
the api returns the project id as string not number

Reply:
Will investigate this

Reply:
changed ids to strings everywhere

Reply:
done